jest.config.js

```diff
@@ -0,0 +1,183 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
Would be nice having some clean up in this file and keep only what's been setup.

Reply:
Yeah, I concur, I just took the template and adapted it (after all, this should once been extended ; ).